wp-content/themes/cpipr/series-single.php

```diff
@@ -134,19 +134,22 @@ function getNextPost($post, $series_map)
 			</div>
 		</div>
 	</div>
+<?php
+if (!empty($next_post)):
+	$heroImageNext = wp_get_attachment_image_src(get_post_thumbnail_id($next_post->ID), 'full');
+	$next_position = array_search($next_post->ID, $series_map);
+?>
 	<div class="wrapper-main-next post-prev" id="wrapper-posts">
 		<div class="wrapper-hero">
-			<?php
-$next_post = getNextPost($post, $series_map);
-if (!empty($next_post)):
-    $heroImageNext = wp_get_attachment_image_src(get_post_thumbnail_id($next_post->ID), 'full');
-    ?>
         <div class="hero-main">
           <div class="wrapper-image" style="background: url('<?php echo $heroImageNext['0']; ?>') no-repeat center/cover;">
             <div class="container-fluid">
               <div class="row-fluid">
                 <div class="span2">
-                  <h3 class="title-post">Siguiente en la serie</h3>
+									<div class="section-title">
+										<h3><?php _e('Next in Series', 'largo'); ?></h3>
+										<?php echo '<h3>' . ($next_position + 1) . ' / ' . count($series_map) . '</h3>'; ?>
+									</div>
```

Comment:
This section should be indented to match the surrounding code.

Reply:
ok, done. please check